staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -2250,7 +2250,7 @@ func TestConsistentList(t *testing.T) {
 	transformer.store = store
 
 	for i := 0; i < 5; i++ {
-		if err := transformer.createObject(); err != nil {
+		if err := transformer.createObject(context.TODO()); err != nil {
```

Comment:
My follow-up will be to change these, at which point only the top-level contexts will need any change.